os/drivers/lwnl/lwnl_evt_queue.h

```diff
@@ -26,7 +26,7 @@
 
 struct lwnl_event;
 struct lwnl_queue;
-int lwnl_add_event(lwnl_cb_status type, void *buffer);
+int lwnl_add_event(lwnl_cb_status type, void *buffer, uint32_t buf_len);
```

Comment:
Could you couple this function declaration with the others below instead of here, with the variable and object declarations please?

Reply:
I'll move the declaration of function. and I'll add description of the functions in another PR later.